### PR TITLE
added OEM external-customer-info

### PR DESCRIPTION
**Pull request by ddubeau**

### CODE REVIEW

File: docs/resources/mws_workspaces.md [hunk 1]
```diff
@@ -183,6 +183,31 @@ output "databricks_token" {
...
+  provider        = databricks.mws
+  account_id      = var.databricks_account_id
+  workspace_name  = local.prefix
+  deployment_name = local.prefix
+  aws_region      = "us-east-1"
+
+  credentials_id           = databricks_mws_credentials.this.credentials_id
+  storage_configuration_id = databricks_mws_storage_configurations.this.storage_configuration_id
+
+  external_custoemr_info {
```
Comment (nfx) [inline docs/resources/mws_workspaces.md:203]: There's a typo. Did you try executing this resource?

File: docs/resources/mws_workspaces.md [hunk 2]
```diff
@@ -183,6 +183,31 @@ output "databricks_token" {
 
 In order to create a [Databricks Workspace that leverages AWS PrivateLink](https://docs.databricks.com/administration-guide/cloud-configurations/aws/privatelink.html) please ensure that you have read and understood the [Enable Private Link](https://docs.databricks.com/administration-guide/cloud-configurations/aws/privatelink.html) documentation and then customise the example above with the relevant examples from [mws_vpc_endpoint](mws_vpc_endpoint.md), [mws_private_access_settings](mws_private_access_settings.md) and [mws_networks](mws_networks.md).
 
+## OEM Term of Service
+
+-> **Note** The following describes how to tag the customer OEM workspace and have the customer sign the terms of service via a click through workflow
```
Comment (nfx) [inline docs/resources/mws_workspaces.md:188]: Is there a public OEM doc?

**Review by nfx — CHANGES_REQUESTED** [review]

File: docs/resources/mws_workspaces.md (hunk 2 above)
Comment (ddubeau) [reply docs/resources/mws_workspaces.md:188]: The doc team haven't published the public OEM doc yet

File: docs/resources/mws_workspaces.md (hunk 1 above)
Comment (ddubeau) [reply docs/resources/mws_workspaces.md:203]: I'll fix the typo and push it again. I have not executed this code.

Comment (ddubeau) [reply docs/resources/mws_workspaces.md]: FYI I've tested and executed the resource with the typo fix.

File: docs/resources/mws_workspaces.md (hunk 2 above)
Comment (nfx) [reply docs/resources/mws_workspaces.md:188]:
```suggestion
-> **Note** The following describes how to tag the customer OEM workspace and have the customer sign the terms of service via a click through workflow. Please contact your Databricks representative for further questions.
```

**Review by nfx — APPROVED** [review]

**Comment by nfx** [comment]
Closing this pull request due to age. Please roll out the new PR once there's a product consesus

**Pull request closed without merging**